docs/blog/2022-03-10-zephyr-3-0-upgrade-prep.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Zephyr 3.0 Update Preparation"
+author: Pete Johanson
+author_title: Project Creator
+author_url: https://gitlab.com/petejohanson
+author_image_url: https://www.gravatar.com/avatar/2001ceff7e9dc753cf96fcb2e6f41110
+tags: [firmware, zephyr, core]
+---
+
+As preperation for completing the [work](https://github.com/zmkfirmware/zmk/pull/1143) to upgrade ZMK to [Zephyr 3.0](https://docs.zephyrproject.org/3.0.0/releases/release-notes-3.0.html), users with user config repositories who wish to avoid future build failures with their GitHub Actions workflows can take steps to adjust
+their repositories now
+
+Existing user config repositories using Github Actions to build will pull down Zephyr 3.0 automatically,
+when the migration of ZMK is completed. However, to be sure the right Docker image is used, you should:
+
+- Open `.github/workflows/build.yml` in your editor/IDE
+- Change `zmkfirmware/zmk-build-arm:2.5` to `zmkfirmware/zmk-build-arm:latest` wherever it is found
```

Comment:
> Existing user config repositories using Github Actions to build will pull down Zephyr 3.0 automatically, when the migration of ZMK is completed.

I think this could be misinterpreted as “existing user config repos will be upgraded automatically" (and therefore no action is required) by someone who is unfamiliar with how things work behind the scenes.

Perhaps it could be reworded to something along the lines of:

```suggestion
GitHub Actions needs to use our latest Docker image to ensure continued compatibility with the ZMK codebase on Zephyr 3.0 (and beyond). You should:

- Open `.github/workflows/build.yml` in your editor/IDE
- Change `zmkfirmware/zmk-build-arm:2.5` to `zmkfirmware/zmk-build-arm:latest` wherever it is found
```

Reply:
Done!